content/docs/introduction/billing.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Billing
+enableTableOfContents: true
+isDraft: true
+---
+
+Neon offers the following plans: **Free Tier**, **Pro**, **Enterprise**, and **Platform Partnership**. The Pro plan is _usage-based_, ensuring you never over-provision and only pay for what you use. The **Enterprise** and **Platform Partnership** plans are custom volume-based plans that offer potential discounts. You can find out more about our [plans](#neon-plans) below.
+
+## Neon billing metrics
+
+Neon's charges for usage based on the following metrics:
+
+- **Compute time**: The amount time that Neon compute resources are active.
+- **Project storage**: The amount of data stored in your Neon projects.
+- **Written data**: The amount of data written from compute to storage.
+- **Data transfer**: The amount of data transferred out of Neon.
+
+See [Billing metrics explained](#billing-metrics-explained) for a detailed description of each metric and how Neon calculates costs.
+
+## Neon plans
+
+|                          | Free Tier                         | Pro (usage-based)             | Custom - Enterprise or Platform Partnership (volume-based)|
+|:-------------------------|:----------------------------------|:-----------------|:--------------------------|
+|**Best for**              | Prototyping and personal projects       | Small-to-medium sized teams, setups with 1 to 3 active databases  | Medium-to-large sized teams, Database fleets, Resale |
+|**Projects**              | 1                                 | 20               | Unlimited                 |
+|**Compute active time per month** | 100 hours                 | Unlimited        | Unlimited                 |
+|**Compute size**          | Up to 1 vCPU with 1 GB RAM   | Up to 7 vCPUs, each with 4 GB RAM     | Custom           |
+|**Storage**               | 3 GB per branch                   | 200 GB        | Unlimited                 |
+|**Dedicated resources**   | -                            | -           | &check;                   |
+|**Project sharing**       | -                            | &check;          | &check;                   |
+|**Auto-suspend compute**  | &check;                           | &check;          | &check;                   |
+|**Configurable auto-suspend compute (coming soon)**  | -                           | &check;          | &check;                   |
+|**Always-on compute (coming soon)**     | -                            | &check;          | &check;                   |
```

Comment:
```suggestion
```
i suggest to drop this item as this is repetitive with the previous item (Configurable auto-suspend compute)